### DISCUSSION TITLE
Problems building bw_web_build.git

### DISCUSSION THREAD

**Discussion by softcongmbh** [dani-garcia/vaultwarden#5285] [Q&A]
Hey everyone,

I want to build bw_web_build (https://github.com/dani-garcia/bw_web_builds.git).

I always get an error: when running the command make docker-extract, I get the following error:

Can you please help me?
![error](https://github.com/user-attachments/assets/1d8a47be-6024-48af-8aed-e32d5d3bd324)

**Comment by stefan0xC** [comment]
What version are you trying to build? From the screenshot I'd assume that you are not using the latest `Dockerfile` (because that would be using [`node:20-bookworm AS build`](https://github.com/dani-garcia/bw_web_builds/blob/847a1c56f49ee0103c631ecdea3685d2363f830e/Dockerfile#L20))

**Comment by softcongmbh** [comment]
Sorry, that was my mistake :D I have now tested it with the latest version, but with the newest version, I get the following error in step 9:
![error](https://github.com/user-attachments/assets/cb1ada30-6f4c-4604-8677-165b4ab8480e)

**Reply by BlackDex** [reply]
It shows `bash\r`. That `\r` is strange, as Linux doesn't use `\r` Nor does macOs in newer versions.
Could it be the files are changed during your git clone/pull?

It looks like the files are changed using incorrect line-endings.
